### ISSUE TITLE
Automate focusable controls standard

### ISSUE THREAD

**Issue by mattwynne** [cucumber-ltd/bbc-a11y#33]
Split off from #11.

See https://github.com/cucumber-ltd/bbc-a11y/blob/master/standards/15_focusable-controls.md

**Comment by mattwynne** [comment]
This looks like it would be hard to build a generic piece of code that could check any page. We might be able to do it with custom extensions so teams could build their own checks per page, or their own list of control elements to be checked.

Needs further analysis / discussion to understand it better.

**Comment by IanPouncey** [comment]
I agree that this is too page specific to automate, but if custom tests could be written it would be of high value as this can be a critical problem.
